components/Header.jsx

```diff
@@ -1,14 +1,15 @@
+import Link from 'next/link';
 import Search from './Search';
 
 export default function Header() {
   return (
     <header>
       <div className="navbar navbar-dark shadow-sm">
         <ul>
-          <li><a href="/">Home</a></li>
-          <li><a href="shops">Shops</a></li>
-          <li><a href="login">Login</a></li>
-          <li><a href="profile">Profile</a></li>
+          <li><Link href="/">Home</Link></li>
+          <li><Link href="/">Shops</Link></li>
```

Comment:
Don't actually need a Shops link in the Header, but this can be a later fix as well.

Reply:
Ok, then the href= should be "shop" not root?

Reply:
Yes, that link for the search page, I am not sure we will what we put in that file, so it is an empty file now.

Reply:
https://nextjs.org/learn/basics/navigate-between-pages/link-component
`<Link href="/posts/first-post">
    <a>this page!</a>
  </Link>`